Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+COPY requirements.txt .
```

Comment:
Ensure that `requirements.txt` exists in the context of the Docker build. If it doesn't, the build will fail at this step.